src/unittests/conversion.spec.ts

```diff
@@ -0,0 +1,34 @@
+export const description = `Unit tests for conversion`;
+
+import { makeTestGroup } from '../common/internal/test_group.js';
+import { float16BitsToFloat32, float32ToFloat16Bits } from '../webgpu/util/conversion.js';
+
+import { UnitTest } from './unit_test.js';
+
+export const g = makeTestGroup(UnitTest);
+
+const cases = [
+  [0b0011110000000000, 1],
+  [0b0000010000000000, 0.00006103515625],
+  [0b0011010101010101, 0.33325195],
+  [0b0111101111111111, 65504],
+  [0b0, 0],
+  [14336, 0.5],
+  [12902, 0.1999512],
+  [22080, 100],
```

Comment:
nit: IMO unless there's a particular reason to use these decimal representations, it would be clearest to use exactly-16-bit long `0b` representations for all of the cases. Also you can use underscore as a separator, like: `0b0011_1100_0000_0000` or even `0b0_01111_0000000000` to show the separation between sign, exponent, and mantissa